### DISCUSSION TITLE
Weekend Bike Action

### DISCUSSION THREAD

**Discussion by MotoUKRider** [devemux86/cruiser#148] [Show and tell]
Why not a picture of the setup, you ask?  This time I remembered to take my spare phone so that I could take pictures of Cruiser "in situ" !

It's a 2nd-gen VStrom 650, using a Cosmo GPS bracket with sprung holder that's available on the 2 big websites beginning with "A".  I've tried the UltimateAddons' case and their locking holder but find this method better for visual navigation as well as ease of removing it for pictures/parking.

I've removed the 2 prongs at the top so that I can slide the phone in from the top without opening the spring too much.

![cruiser-bike-1](https://user-images.githubusercontent.com/125696109/235355197-3116cb32-3631-4202-85af-c52878ba1960.jpg)

![cruiser-bike-2](https://user-images.githubusercontent.com/125696109/235355202-4723ebd2-c927-4e66-92dc-a2524d831883.jpg)

**Comment by devemux86** [comment]
@MotoUKRider thanks for using Cruiser and the very nice pictures! :+1:
